libqtile/window.py

```diff
@@ -195,8 +197,21 @@ def __init__(self, window, qtile):
         }
         self.update_hints()
 
-    x = property(fset=_geometry_setter("x"), fget=_geometry_getter("x"))
-    y = property(fset=_geometry_setter("y"), fget=_geometry_getter("y"))
+    @property
+    def x(self):
+        return _geometry_getter("x")(self)
+
+    @x.setter
+    def x(self, value):
+        _geometry_setter("x")(self, value)
+
+    @property
+    def y(self):
+        return _geometry_getter("y")(self)
+
+    @y.setter
+    def y(self, value):
+        _geometry_setter("y")(self, value)
```

Comment:
You prefer it this way!?

Reply:
ack, sorry, remnant of another attempt at solving this

Reply:
Doesn't this way require resolving the {s,g}etter every time, where before it was resolved upon class definition?

Reply:
yes

Reply:
I'm going to add a commit that gives width and height the same treatment